### PR TITLE
DOCS: Create requirements.txt

### PR DESCRIPTION
**Pull request by cleobnvntra**
Closes issue #2 

# Description
- Created requirements.txt based on requirements mentioned in README.md

# How I created requirements.txt
- Installed the required packages manually using `uv pip install <package>`
- Created `requirements.txt` using the command `uv pip freeze > requirements.txt`

# How to install packages in requirements.txt using `uv`
- Run using the command `uv pip install -r requirements.txt` to install packages listed in requirements.txt

### CODE REVIEW

**Comment by surreal30** [comment]
Can you add this step in `README.md` as well?

**Comment by cleobnvntra** [comment]
> Can you add this step in `README.md` as well?

Sure thing!

**Comment by cleobnvntra** [comment]
I have made the push to include the steps needed to install using `requirements.txt` and how to update `requirements.txt` when new packages are installed!

File: README.md [hunk 1]
```diff
@@ -1,23 +1,45 @@
...
+uv pip freeze > requirements.txt
+```
 
 ---
+
 I am trying to document this journey and other cool tech stuff! Find it here
 Twitter: [@SurrealDotTxt](https://x.com/SurrealDotTxt)
 Newsletter: [https://tilincode.substack.com](https://tilincode.substack.com/)
-Made with love by Surreal \^_^
+Made with love by Surreal \^\_^
```
Comment (surreal30) [inline README.md:45]: Nit: This doesn't seem to make a difference on the README.

Otherwise looks good.

**Review by surreal30 — APPROVED** [review]
Looks good. Merging.

**Pull request merged**